### ISSUE TITLE
[Security] anonymous: lazy deprecation warning

### ISSUE THREAD

**Issue by rcapile** [symfony/symfony-docs#13749]
I upgrade to symfony/security-bundle to 5.1 and I got the deprecation warning `Since symfony/security-bundle 5.1: Using "anonymous: lazy" to make the firewall lazy is deprecated, use "anonymous: true" and "lazy: true" instead.` 

But there's nothing about this change in https://symfony.com/doc/current/security.html#a-authentication-firewalls

Sorry I can't make a PR because I believe it would require some explanation why it's now two options and a "New in Version 5.1" box, right?

**Comment by yceruto** [comment]
The doc needs to be update yes, but the recipe was already updated https://github.com/symfony/recipes/pull/778, so you can fix this warning by executing `composer sync-recipes --force`

**Comment by wouterj** [comment]
Oh sorry for making the change and completely ignoring updating docs and recipes. Thanks @yceruto and @rcapile for taking care of that!

> Sorry I can't make a PR because I believe it would require some explanation why it's now two options and a "New in Version 5.1" box, right?

Please feel free to submit a PR. I don't think we need to explain anything here, a "new in 5.1" is usefull though. You can add it using this syntax:

```rst
.. versionadded:: 5.1

    The ``lazy: true`` option was introduced in Symfony 5.1. Prior to version 5.1,
    it was enabled using ``anonymous: lazy``
```

**Comment by rcapile** [comment]
done

**Comment by javiereguiluz** [comment]
Fixed in #13750. Thanks!
